workflows/seqinspector.nf

```diff
@@ -88,7 +88,7 @@ workflow SEQINSPECTOR {
     if (!("fastqscreen" in skip_tools)) {
```

Comment:
Admittedly, `params.fastq_screen_references` is defined both in the main `nextflow.config` and the _test_ profile with a default, so it is unlikely to happen unintentionally, but if somebody were to set it to `null` in a param file, it would crash the pipeline otherwise?

```suggestion
    if (!("fastqscreen" in skip_tools) &&  params.fastq_screen_references) {
```

Reply:
I'm thinking that explicitly setting the parameter to null without explicitly skipping the tool would be a pretty good reason to crash 🤔 If we start to have multiple different ways tools can be skipped it introduces ambiguity which maybe we want to avoid for now.

I tried adding some assertions in [0ba1652](https://github.com/nf-core/seqinspector/pull/77/commits/0ba165203724f077c859f6f27bf0b8c101250670), but then tried reverting them when I saw some cryptic CI failures. They still fail even on net-0 change code, so it's likely something outside the scope of this PR 😅